boards/opencm904/include/board.h

```diff
@@ -55,11 +55,20 @@ extern "C" {
 /** @} */
 
 /**
- * @brief   User button
+ * @brief User button
  */
 #define BTN_B1_PIN          GPIO_PIN(PORT_C, 15)
 
 /**
+ * @brief Pin used to switch RX and TX mode for the Dynamixel TTL bus
```

Comment:
This block of code doesn't seem to be related to this change but rather to #6725. Can you revert it and backport to the related PR ?